docs/Apps/Metabolomic Data/Labeled LC-MSMS Workflow.md

```diff
@@ -139,6 +141,86 @@ Click "Run* to plot the fractional enrichment plot with all the labels present a
 
 ![Visualization plot](../../img/LabeledLC-MS2Workflow/VisualizationPlot.png) <center> **Figure 16.** Visualization plot </center>
 
+## Phi<sup>BETA</sup> Tab
+
+This tab enables the user to perform Phi analysis. The term 'Phi' denoted by 'φ' is ratios of fractional enrichments of isotopomers/isotopologues of a product to the fractional enrichments of isotopomers/isotopologues of a precursor in the pathway.
```

Comment:
@KMaheswari We do not mention "user" anywhere. We refer to the person reading this directly. Rephrase to "This tab enables you..."

Reply:
Also, shouldn't it be "ratio" instead of "ratios"?